pyproject.toml

```
@@ -21,7 +21,7 @@ classifiers = [
 repository = "https://github.com/numaproj/numaflow-python"
 
 [tool.poetry.dependencies]
-python = ">=3.9, <3.13"
+python = ">=3.9,<3.14"
```

Add `"Programming Language :: Python :: 3.12"`
in the classifiers

---

You also need to add tests to make sure python 3.13 pass
https://github.com/numaproj/numaflow-python/blob/main/.github/workflows/run-tests.yml

---

Per my other comment, please consider changing to:
```suggestion
python = "^3.9"
```

Or if none of this packages dependencies cap the Python version (unlikely), then this would be better:
```suggestion
python = ">=3.9"
```